README.md

```diff
@@ -405,6 +405,148 @@ Impact layers are computed by combining a raster layer with a vector layer based
 To display additional metadata about a layer, you can add a `content_path` attribute to any layer. The attribute expects a path to a `.md` or `.html` file that is stored in `public/data/${REACT_APP_COUNTRY}/filename.ext` directory. For example: `public/data/myanmar/contents.md`
 The application will show an icon next to the layer in the legend if this attribute is configured, and will display the content in a modal window if the icon is clicked.
 
+## Dashboards
+
+Dashboards are customizable reports that combine maps, charts, tables, and text blocks in a flexible layout. They are configured in `prism.json` under the `configuredReports` array.
+
+### Dashboard Configuration Structure
+
+Each dashboard is defined as an object with the following properties:
+
+```json
+{
+  "title": "Dashboard Title",
+  "path": "url-friendly-path",
+  "isEditable": true,
+  "firstColumn": [...],
+  "secondColumn": [...],
+  "thirdColumn": [...]
+}
+```
+
+**Dashboard-level properties:**
+- `title` (required): Display name of the dashboard
+- `path` (optional): URL path for the dashboard. If omitted, it will be auto-generated from the title
+- `isEditable` (optional): If `true`, users can edit the dashboard in the UI. Defaults to `false`
+- `firstColumn`, `secondColumn`, `thirdColumn` (required): Arrays of dashboard elements. At least `firstColumn` must be defined
+
+### Dashboard Elements
+
+Each column contains an array of elements. There are four types of elements:
+
+#### 1. MAP Element
+
+Displays an interactive map with pre-selected layers.
+
+```json
+{
+  "type": "MAP",
+  "defaultDate": "2025-04-01",
+  "mapPosition": "left",
+  "minMapBounds": [31, -25, 40, -11],
+  "title": "Temperature Anomaly Map",
+  "preSelectedMapLayers": [
+    { "layerId": "lst_day_anomaly", "opacity": 0.7 },
+    { "layerId": "rainfall_dekad", "opacity": 0.8 }
+  ]
+}
+```
+
+**Properties:**
+- `type`: Must be `"MAP"`
+- `preSelectedMapLayers` (required): Array of layer objects to display on the map
+  - `layerId`: Layer ID from `layers.json`
+  - `opacity` (optional): Layer opacity (0.0 to 1.0). Defaults to 1.0
+- `defaultDate` (optional): Initial date for the map in `YYYY-MM-DD` format
+- `mapPosition` (optional): `"left"` or `"right"` - used for side-by-side map comparison

```

Comment:
Not specific to the README, but I think we ought to get rid of this config option with the new layout structure.